Algorithm-Spinal-Tap-Case.md

```diff
@@ -1,30 +1,43 @@
 # Algorithm Spinal Tap Case
 
-### Problem Explanation:
+:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](FreeCodeCamp-Get-Help) if you get stuck. Try to pair program :busts_in_silhouette: and write your own code :pencil:
 
-- Convert the given string to an all lowercase sentence joined by dashes.
+### :checkered_flag: Problem Explanation:
 
-## Hint: 1
+Convert the given string to an all lowercase sentence with words joined by dashes.
```

Comment:
~~Convert the given string to an all lowercase sentence with words joined by dashes.~~ Convert the given string to a lowercase sentence with words joined by dashes.